### PR TITLE
contrib: Add a tool to analyze local Docker images

### LINKED ISSUE #15: Produce a script to analyze local images easily

### PR DESCRIPTION
**Pull request by Quentin-M**
Fixes #15

### CODE REVIEW

File: contrib/analyze-local-images/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Analyze local images
+
+This is a basic tool that allow you to analyze your local Docker images with Clair.
+It is intended to let everyone discover Clair and offer awareness around containers' security.
+There is absolutely no guarantee and it only uses a minimal subset of Clair's features.
```
Comment (josephschorr) [inline contrib/analyze-local-images/README.md]: no guarentees

Comment (Quentin-M) [inline contrib/analyze-local-images/README.md]: no guarantees* ?

Comment (josephschorr) [inline contrib/analyze-local-images/README.md]: "There are absolutely no guarantees and it only uses..."

**Comment by jzelinskie** [comment]
LGTM

**Pull request merged**